app/models/manageiq/providers/hawkular/datawarehouse_manager/event_catcher/runner.rb

```diff
@@ -45,12 +45,15 @@ def event_to_hash(event, current_ems_id)
     event = event.dup
     event.severity = map_severity(event.severity) # gets into event.full_data
     event.url = event.tags[TAG_URL]
+    event.ems_ref = event.id
+    event.resolved = %w(RESOLVED).include?(event.status)
```

Comment:
```ruby
event.status == "RESOLVED"
```
Or even better, implementing `resolved?` on the event itself, and then youn can use it on the timestamp as well.